### ISSUE TITLE
Localhost XAMP 

### ISSUE THREAD

**Issue by raoharis951** [givanz/VvvebJs#338]
It is not running on localhost I tried alot & also i tried on hostinger too the them section is empty. Will you please provide me the complete guided video of the process to run on localhost.

**Comment by raoharis951** [comment]
Failed to load resource: the server responded with a status of 404 (Not Found) 
This is the error that appears in Console and i want to run it on local.

**Comment by raoharis951** [comment]
Will you please make a video of complete installation for admin dashoard and editor how to connect them also let me know the whole process for this step by step.

**Comment by givanz** [comment]
> admin dashboard 

Do you use Vvveb CMS or VvvebJs page builder? 

Only the CMS has an admin, VvvebJs has only one page, the page builder it comes without an admin dashboard.

Vvveb CMS has VvvebJs page builder built in you don't need to install both.

Do you need a video with the CMS installation or after installation setup? 

Installation is very easy, after unzipping latest.zip  to htdocs folder follow the installation instructions https://docs.vvveb.com/installation 

You can use SQLite if you don't want to configure database details for an easier install.

**Comment by raoharis951** [comment]
Yes I want CMS complete video of this project both installation and after installation  so i can understand how it run.

**Comment by INTERC3PTI0N** [comment]
> ![CaptureERROR](https://private-user-images.githubusercontent.com/57353650/306896607-8886077f-4599-47ad-afc8-aa60fa991a7c.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkzMDExNzYsIm5iZiI6MTcwOTMwMDg3NiwicGF0aCI6Ii81NzM1MzY1MC8zMDY4OTY2MDctODg4NjA3N2YtNDU5OS00N2FkLWFmYzgtYWE2MGZhOTkxYTdjLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzAxVDEzNDc1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgxNGM4Yjk5MmEzMTZjNGExN2YyNzAyYzRmNTM1MGZkMDFkMzYxNDNiYjRiYTlmNmE1YTRiYzAxYWUyNzIyMmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gF60lBMwWnFYgIEkNzn9IUnSk3gSPdwL1_ZJRN8joWA) Yes I want CMS complete video of this project both installation and after installation so i can understand how it run.

I initially faced the same issue too, but it seems in order to make it work you have to put all the files into the main htdocs directory itself, not in a sub-directory inside htdocs. I just installed a copy of xampp and deleted everything from htdocs folder, and extracted the zip file directly inside htdocs. That works well.

**Comment by givanz** [comment]
@INTERC3PTI0N 

Thanks for info on subdirectory issues with XAMPP.
For subdirectory installation `V_SUBDIR_INSTALL` must be set in `env.php` to subdirectory name

```php
defined('V_SUBDIR_INSTALL') || define('V_SUBDIR_INSTALL', '/subdir');
```

https://github.com/givanz/Vvveb/blob/master/env.php#L46-L47 

But is better to use from htdocs directly because with subdirectory install there can be issues with things like multi site.


@raoharis951

I just saw the screenshot, it was not visible before, I see that you are loading Vvveb from  `/latest`  subfolder you can either  set `V_SUBDIR_INSTALL` or move the files to `/htdocs` as @INTERC3PTI0N recommends.

```php
defined('V_SUBDIR_INSTALL') || define('V_SUBDIR_INSTALL', '/latest');
```

**Comment by hansana2** [comment]
I moved everything to the /htdocs and it worked. but I would like to keep everything organized with a sub directory. Editing the `env.php` file did not worked, and as you mentioned can you fix the issues with things like multi site?

**Comment by DevHanza** [comment]
I passed 2 Installation screens and now it's showing me a error, after I redirected to the dashboard. 

Here is a screenshot. What am I doing wrong? (I'm not using any subdirectory.)

![Image](https://github.com/user-attachments/assets/ebeab871-d8e8-4062-b18b-c43e2dfc245b)
